File: tests/corpus/planted/python/tls_config.py
"""Planted fixture: legacy TLS/SSL protocol constants (B1).

Every reference below is a known-detectable protocol constant (see
tests/corpus/planted/expected.yaml). Not real code — do not import.
"""

import ssl

protocol_tlsv1 = ssl.PROTOCOL_TLSv1
protocol_tlsv1_1 = ssl.PROTOCOL_TLSv1_1
protocol_sslv3 = ssl.PROTOCOL_SSLv3
tlsversion_tlsv1 = ssl.TLSVersion.TLSv1
tlsversion_tlsv1_1 = ssl.TLSVersion.TLSv1_1
tlsversion_sslv3 = ssl.TLSVersion.SSLv3

File: tests/corpus/planted/go/tls_config.go
// Planted fixture: legacy TLS protocol versions and static-RSA cipher
// suites (B1).
//
// Not real code — do not build as part of any module. See
// tests/corpus/planted/expected.yaml.
package planted

import "crypto/tls"

func plantedTLSConfig() {
	_ = &tls.Config{MinVersion: tls.VersionTLS10}
	_ = &tls.Config{MaxVersion: tls.VersionTLS11}
	_ = tls.VersionSSL30
	_ = tls.TLS_RSA_WITH_RC4_128_SHA
	_ = tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA
	_ = tls.TLS_RSA_WITH_AES_128_CBC_SHA
	_ = tls.TLS_RSA_WITH_AES_256_CBC_SHA
	_ = tls.TLS_RSA_WITH_AES_128_CBC_SHA256
	_ = tls.TLS_RSA_WITH_AES_128_GCM_SHA256
	_ = tls.TLS_RSA_WITH_AES_256_GCM_SHA384
}
